data/basics/teal.go

```diff
@@ -23,6 +23,10 @@ import (
 	"github.com/algorand/go-algorand/config"
 )
 
+// MaxLogCalls is the highest allowable log messages that may appear in
+// any version, used only for decoding purposes.
+const MaxEncodedTealLogCalls = 32
```

Comment:
should you note that this value should never be decreased?

Reply:
Right, done!